pyproject.toml

```diff
@@ -42,11 +42,13 @@ skorch = "0.11.0"
 torch = "1.13.1"
 typing-extensions = "4.5.0"
 brevitas = "0.8.0"
-onnxoptimizer = "0.3.10"
-onnxruntime = "1.13.1"
+onnx = "1.15.0"
```

Comment:
the only changed I've made is in this file, the rest is just conformance/pcc , lock file or licenses